gapic/src/main/com/google/gapid/widgets/ImagePanel.java

```diff
@@ -343,7 +343,9 @@ private void setAlphaEnabled(boolean enabled) {
 
   @Override
   public void startLoading() {
-    if (imageComponent.isOpenGL()) {
+    // Avoid switching to 'loading' image for framebuffer views --
+    // the loading image makes it difficult to perceive image changes
+    if (imageComponent.isOpenGL() && (analyticsView != View.Framebuffer)) {
```

Comment:
I don't think we want to do this. `LoadablePanel` already has a delay built in. If it `stopLoading()` is called within 0.5s of `startLoading()`, the loading indicator is never shown.
I think the true solution here is finding out why it takes so much time to load the image if it's already in the cache.

Reply:
Loading attachment images takes multiple seconds in large traces.  Once they have been loaded the internals of `LoadablePanel` cause the image area to be cleared which creates a 'flash' even if the loading indicator is not shown.